### PR TITLE
Build with CakeHax.

### PR DESCRIPTION
**Pull request by mid-kid**
Made some changes to the makefile, adding a new target to build Decrypt9 with CakeHax.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -138,7 +145,7 @@ $(OUTPUT).elf	:	$(OFILES)
 
 #---------------------------------------------------------------------------------
 %.bin: %.elf
-	@$(OBJCOPY) -O binary $< $@
+	@$(OBJCOPY) --set-section-flags .bss=alloc,load,contents -O binary $< $@
```
Comment (archshift) [inline Makefile:148]: Does this effect anything for the gateway or bootstrap loaders?

Comment (b1l1s) [inline Makefile:148]: It does affect gateway, there's no second stage ROP for spider, so you need to use the one made specifically for CakeHax. e.g. http://dukesrg.no-ip.org/3ds/cakes?Decrypt9.dat

As for mset entry point, users have to use [CakesROP](https://github.com/mid-kid/CakesROP) to install our version of the first stage NVRAM exploit too (supports all regions and downgraded mset).

I'm not sure about bootstrap. @mid-kid can probably answer that.

Comment (mid-kid) [inline Makefile:148]: Nevermind what @b1l1s said above, as he didn't read the question right.
What that line does, is it appends the bss section to the binary.
It shouldn't have any effect on the Gateway loader, nor bootstrap.

**Comment by archshift** [comment]
Superseded by #39.

**Pull request closed without merging**